### ISSUE TITLE
`nvm` charter review

### ISSUE THREAD

**Issue by ljharb** [openjs-foundation/cross-project-council#750]
The final item on `nvm`'s [onboarding checklist](https://github.com/openjs-foundation/project-status/issues/2) is CPC approval of the project charter (this issue was requested in today's [meeting](https://github.com/openjs-foundation/cross-project-council/issues/748)).

This commit (on my personal fork) will be pushed to the main nvm repo soon; in the meantime, it's ready for CPC review: https://github.com/ljharb/nvm/blob/HEAD/PROJECT_CHARTER.md

@openjs-foundation/cpc: please comment with any feedback, or please 👍 -react to the OP to indicate approval.

**Comment by bnb** [comment]
provided some feedback, LGTM.

**Comment by mhdawson** [comment]
Seems ok to me.

**Comment by tobie** [comment]
This is simple and to the point. I would encourage you to simplify the language around "The Team," "Maintainers," and "nvm's Team Governing Body" which all point to the same entity and just use "Maintainers" everywhere. Additionally, I'd recommend also defining that term in section 5.

**Comment by ljharb** [comment]
@tobie i can certainly use the same term; my hope was to not need to change the charter in the future when/if i get more maintainers. Thoughts?

**Comment by tobie** [comment]
> my hope was to not need to change the charter in the future when/if i get more maintainers.

I don't think you're making a distinction in the charter between _The Team_ and _The Maintainers_, so that really shouldn't be an issue as long as you make sure to always pluralize _Maintainers_.

**Comment by eemeli** [comment]
Overall looks good, but the definitions of "Contributors", "Maintainers" and "Team" is a bit unclear. Not because they're not well defined, but because they're defined in multiple places. I'd second @tobie on dropping "Team" entirely, and just referring to "Maintainers", as that appears to be intended to be synonymous. Especially as the alternative understanding is that "Team" also includes "Contributors", and that appears to be almost 300 people so far.

Additionally, where are the "Maintainers" defined? If the text of README.md and GOVERNANCE.md were to disagree, which would be considered the canonical source? It might be easier to update the Section 2 & 3 "team" or "maintainers" links to also point at GOVERNANCE.md, leaving the "Team" section of the README.md more clearly as just informative.

It might also be easier to refer to GOVERNANCE.md in Section 5 for the "Maintainer" and "Contributor" definitions, as that would avoid the current slight difference between the definitions of that term in the two documents.

But all that is nitpicking, tbh. It does look good, especially for a one-maintainer project!

**Comment by SaraJo** [comment]
LGTM. Agreed on just maintainers being more intuitive. (or Maintainer).

**Comment by ljharb** [comment]
Thanks for everyone's feedback!

Contributors are widely understood to be "anyone whose work is included in the project" (and are explicitly defined already in the charter), and "team"/"maintainers" to me explicitly means "people with decision-making authority for the project". I'll definitely consolidate those terms; thanks for the feedback. I've also added "Maintainers" to the definitions list in the charter alongside "contributors".

@eemeli The charter points at the README, thus that's the source of truth for "who is a Maintainer". GOVERNANCE is for describing the roles and responsibilities of Maintainers, not their identities (per the relevant section in the charter).

I've updated the branch; any additional thoughts?
